src/routes/component/accordion/marko+page.marko

```diff
@@ -0,0 +1,23 @@
+import metadata from "./metadata.json";
+
+<component-tabs componentInfo=metadata currentTab="Marko">
+    <div id="accordion-marko">
+        <h1>Accordion Marko Component</h1>
+        <p>
+            The Accordion Marko component is a responsive, accessible, and high-performance UI element designed for modern Marko applications. Optimized for server-side rendering and streaming, it ensures fast load times and smooth user interactions. Built with semantic HTML and robust accessibility support, the component offers full keyboard navigation, logical focus states, and screen reader compatibility.
+        </p>
+        <p>
+            With Evo Web's Marko implementation, developers get an accordion that adapts seamlessly across devices and themes, while maintaining consistency with CSS and React versions. Whether you're building simple forms or complex interactive interfaces, the Accordion Marko component provides a reliable, standards-aligned foundation that enhances both usability and performance.
```

Comment:
Is this going to be repeated for each page? This seems like generic text which can be used on any component and doesn't really add much.